package.json

```diff
@@ -33,7 +33,7 @@
     "httpntlm": "1.7.5",
     "inherits": "2.0.3",
     "lodash": "4.17.4",
-    "postman-collection": "1.2.9",
+    "postman-collection": "https://github.com/postmanlabs/postman-collection#feature/support-multiple-protocols",
```

Comment:
waiting for the [PR](https://github.com/postmanlabs/postman-collection/pull/400) to get merged
once the sdk is released will update the version